src/components/networks/networkOverviewTab.jsx

```diff
@@ -149,3 +192,93 @@ export const NetworkOverviewTab = ({ network }) => {
 NetworkOverviewTab.propTypes = {
     network: PropTypes.object.isRequired,
 };
+
+const NetworkAddStaticHostEntries = ({ idPrefix, network, parentIndex }) => {
+    const [isOpen, setIsOpen] = useState(false);
+    const [isSubmitted, setIsSubmitted] = useState(false);
+    const [ipAddress, setIpAddress] = useState('');
+    const [macAddress, setMacAddress] = useState('');
+    const [error, setError] = useState();
+
+    const add = () => {
+        setIsSubmitted(true);
+
+        if (!ipAddress || !macAddress)
+            return;
+
+        return networkAddStaticHostEntries({
+            connectionName: network.connectionName,
+            objPath: network.id,
+            macAddress,
+            ipAddress,
+            parentIndex,
+        })
+                .then(
+                    () => {
+                        setIsOpen(false);
+                        setIpAddress('');
+                        setMacAddress('');
+                        setError(undefined);
+                        networkGet({ connectionName: network.connectionName, id: network.id, updateOnly: true });
+                    },
+                    exc => setError(exc.message)
+                );
+    };
+
+    let addEntry = (
+        <Button variant="link"
+                id={idPrefix + "-static-host-entries-add"}
+                isAriaDisabled={network.active}
+                isInline
+                onClick={() => setIsOpen(true)}>
+            {_("add entry")}
+        </Button>
+    );
+    if (network.active) {
+        addEntry = (
+            <Tooltip content={_("Deactivate the network in order to edit it")}>
+                {addEntry}
+            </Tooltip>
+        );
+    }
```

Comment:
Why cannot you edit network while it's active? the Update API has flags for that: https://libvirt.org/html/libvirt-libvirt-network.html#virNetworkUpdateFlags
And I think it's a common use case to add host entries to an active network without the need to destroy it and activate it again?